clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -133,6 +133,16 @@ std::string getDefaultProfileGenName() {
              : "default_%m.profraw";
 }
 
+// Path and name of file used for profile generation
+std::string getProfileGenName(const CodeGenOptions &CodeGenOpts) {
```

Comment:
Move outside of the anonymous namespace and use `static` https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions

Reply:
sure. FYI I was following the style used by getDefaultProfileGenName above

Reply:
done